stress-job.groovy

```diff
@@ -5,7 +5,7 @@ pipelineJob("Stress-job") {
         stringParam('BRANCH', '', 'James branch')
         stringParam('SIMULATION', '', 'Gatling simulation')
         stringParam('GITHUB_SIMULATIONS', '', 'Github repository containing gatling simulations')
-        stringParam('PROFILE', 'reference', 'Blobs store target: reference (default, for over-time performance tracking), swift or s3')
+        stringParam('PROFILE', 'reference', 'Blobs store target: reference (default, for over-time performance tracking), s3 or s3-local')
```

Comment:
* `refrence` will be back as soon as the provisioning is finished
* `s3` will be like the Mokas did before, to some distant s3 ovh object storage... I will need to figure out how to access this
* `s3-local` : I added this to run a virgin s3 object-storage locally

Reply:
s3 needs some environment variable setup apparently

Reply:
That's supposed to be done by the block I put back in the groovy stress test file in https://github.com/linagora/james-project/pull/3811/files#diff-586cbbd770bf7627b91cbdf94e9239baR48 (and that's what we should test and fix)